File: introduction/ITP1_1_D-Watch.php
<?php
$input_lines = fgets(STDIN);
$time = trim($input_lines);
$hour = floor($time / 3600);
$minutes = floor(($time / 60) % 60);
$seconds = floor($time % 60);
echo $hour . ':' . $minutes . ':' . $seconds;
?>

File: introduction/ITP1_1_D-Watch.cpp
#include <iostream>
using namespace std;

int main(void) {
  int time;
  cin >> time;
  int hour = time / 3600;
  int minutes = (time / 60) % 60;
  int seconds = time % 60;
  cout << hour << ':' << minutes << ':' << seconds << endl;
  return 0;
}
